ccd-definition/ComplexTypes/CareSupervision/HearingBooking.json

```diff
@@ -117,7 +117,7 @@
     "HintText": "Use 24 hour format",
     "SecurityClassification": "Public",
     "FieldShowCondition": "endDateDerived=\"No\"",
-    "Searchable": "N"
+    "Searchable": "Y"
```

Comment:
just wondering what's the usage of `Searchable`?

Reply:
Ah if it is "Y" then it is searchable using elastic search, so will need this one for the batch job to search based on the hearing end date!